cli/uninstall/uninstall.go

```diff
@@ -173,3 +186,93 @@ func GetList(cliOpts *config.CliOpts) []string {
 
 	return list
 }
+
+// searchLatestVersion searches for the latest installed version of the program.
+func searchLatestVersion(linkName, binDst, headerDst string) (string, error) {
+	var programsToSearch []string
+	if linkName == "tarantool" {
+		programsToSearch = []string{search.ProgramCe, search.ProgramEe}
+	} else {
+		programsToSearch = []string{linkName}
+	}
+
+	programRegex := regexp.MustCompile(
+		"^" + progRegexp + version.FsSeparator + verRegexp + "$",
+	)
+
+	binaries, err := os.ReadDir(binDst)
+	if err != nil {
+		return "", err
+	}
+
+	latestVersionInfo := version.Version{}
+	latestVersion := ""
+
+	for _, binary := range binaries {
```

Comment:
Do we check here if entry is not a directory?

Reply:
Fixed. Now check it inside cycle.